client/js/place.js

```diff
@@ -456,6 +457,34 @@ var place = {
         this.didSetHash = true;
     },
 
+    contextMenu: function(event) {
+        event.preventDefault();
+        let zoom = place._getZoomMultiplier();
+        let x = Math.round((event.pageX - $(place.cameraController).offset().left) / zoom)
+        let y = Math.round((event.pageY - $(place.cameraController).offset().top) / zoom)
+        return $.get(`/api/pixel?x=${x}&y=${y}`).done(data => {
+            if(data.pixel == null) data.pixel = { // No pixel was clicked... For simplicity we just change the response
```

Comment:
Especially not making up fake data, as we may and probably will make changes to this undocumented API